django/contrib/gis/db/backends/oracle/introspection.py

```diff
@@ -24,12 +21,11 @@ def get_geometry_type(self, table_name, geo_col):
                     (table_name.upper(), geo_col.upper())
                 )
                 row = cursor.fetchone()
-            except Exception as msg:
-                new_msg = (
+            except Exception:
+                raise Exception(
```

Comment:
I think in cases like this where a more helpful error message is raised, the `from exc` syntax should be used so that "The above exception was the direct cause of the following exception:" occurs rather than "During handling of the above exception, another exception occurred:". That's my understanding of the difference.
https://docs.python.org/3/reference/simple_stmts.html#raise